CONTRIBUTING.md

```diff
@@ -95,15 +96,20 @@ You can check your setup by running `yarn test`, it should test all packages and
 
 ## Releasing
 
-1. Update `packages/apollo-angular/CHANGELOG.md`
-2. Run something like that:
-```sh
-VERSION=1.2.3
-./scripts/version.js $VERSION
-yarn run deploy
-git add .
-git commit -m "Release v$VERSION"
-git tag $VERSION -m $VERSION
-git push
-```
-3. Create a release on https://github.com/kamilkisiela/apollo-angular/releases/new
+We are using `Changeset` to manage and publish our releasing flow.
```

Comment:
Over kill it?

Reply:
I think it's fine, but you can also just link to `changeset` repo and people will find there instructions 😆